source/appModuleHandler.py

```diff
@@ -112,12 +112,14 @@ def _warnDeprecatedAliasAppModule() -> None:
 	except KeyError:
 		raise RuntimeError("This function can be executed only inside an alias App Module.") from None
 	else:
-		log.warning(
-			(
-				f"Importing from appModules.{currModName} is deprecated,"
-				f" you should import from appModules.{replacementModName}."
-			)
+		importFailMsg = (

```

Comment:
`Fail` implies that it won't allow the import.
```suggestion
		deprecatedImportWarning = (

```